README.md

```diff
@@ -36,6 +36,7 @@ This is list of known compatible USB hubs:
 | Basler             | 2000036234                                           | 4     | 3.0 |`0451:8046`| 2016    |      |
 | Belkin             | F5U101                                               | 4     | 2.0 |`0451:2046`| 2005    | 2010 |
 | BenQ               | PD2700U 4K Monitor (works only in USB2 mode)         | 4     | 3.1 |`05E3:0610`| 2018    |      |
+| BenQ               | PD3220U                                              | 7     | 2.1 |`05E3:0610`| 2019    |      |
```

Comment:
Why only USB 2.1? It sure must support USB 3?

Reply:
Can you please edit this to mention only 4 ports? And USB 3.1. Thanks!

Reply:
I appended the modifications in a new commit, should be ok I hope? :)